modules/aws-iam-policies/main.tf

```diff
@@ -23,10 +24,19 @@ data "aws_iam_policy_document" "emr_creator_policy" {
       "elasticmapreduce:TerminateJobFlows",
       "iam:PassRole"
     ]
-    resources = [
-      "arn:${var.arn_partition}:iam::${data.aws_caller_identity.current.account_id}:role/*",
-      "arn:${var.arn_partition}:elasticmapreduce:*:${data.aws_caller_identity.current.account_id}:cluster/*"
-    ]
+    resources = flatten([
+                  length(var.tamr_emr_role_arns)==0 ?
+                      ["arn:${var.arn_partition}:iam::${data.aws_caller_identity.current.account_id}:role/*"] :
+                      [ for emr_role_arn in var.tamr_emr_role_arns :
+                        emr_role_arn
+                      ],
+                  length(var.tamr_emr_cluster_ids)==0 ?
+                      ["arn:${var.arn_partition}:elasticmapreduce:${data.aws_region.current.name}:${data.aws_caller_identity.current.account_id}:cluster/*"] :
```

Comment:
perhaps EMR module should have the EMR ARN as an output similar to the role ARNs

Reply:
Yes, I'm disappointed that it doesn't already.